newApp.html

```diff
@@ -0,0 +1,160 @@
+---
+layout: no-header-no-footer
+permalink: /newapp-local
+---
+<!-- Stylesheet -->
+<link rel="stylesheet" type="text/css" href="newAppStyles.css">
+
+<!-- Jumbotron-->
+<div class="jumbotron jumbo-viewport" style="background-image: url('https://crds-media.imgix.net/13kNRYdTJFx5IegWEJSi4P/99d334906bfbfbe8995ead92d236cae3/ag-bg.jpg');">
+  <div class="flex-header soft-quarter-top header crds-shared-header">
+    <a id="crds-shared-header-logo" href="https://crossroads.net/" class="navbar-brand logo logo-margin">
+      <img src="https://crds-media.imgix.net/1V5PIAPuVS9ttWnmMQ2oXM/d25f0d77bcbaa327cfe8afda0e11ecdf/logo-crds-church.svg">
+    </a>
+    <div class="app-store-buttons hide-buttons">
+      <a href="https://itunes.apple.com/us/app/crossroads-anywhere/id1029478803?mt=8">
+        <img src="https://crds-media.imgix.net/7c7KaoRDGPkZZogoDt9AwJ/e4a1d27f49d75eb39be53f993cab647a/apple-button.svg" alt="Apple App Store" class="app-button" style="margin-right: 15px;" title="Apple App Store" />
+      </a>
+      <a href="https://play.google.com/store/apps/details?id=net.crossroads.crossroads&amp;hl=en">
+        <img src="https://crds-media.imgix.net/7dFtrIsd22DHPYkudlXC4C/bac32e1de443d0765efcbda9f1f3e051/google-button.svg" alt="Google Play" class="app-button" style="margin-right: 30px;" title="Google Play" />
+      </a>
+    </div>
+  </div>
+  <!-- Crossroads Anywhere & Tagline-->
+  <div class="container title-top">
+    <div class="row">
+    <p class="font-family-condensed text-uppercase">Crossroads Anywhere App</p>
```

Comment:
```suggestion
    <p class="font-family-condensed text-uppercase font-size-small">Crossroads Anywhere App</p>
```

This text gets large on mobile.